js/paper.js

```diff
@@ -39,7 +39,9 @@ chrome.contextMenus.onClicked.addListener(function(info, tab) {
 });
 
 function authorise() {
-  //TODO: create new tab to request authorisation (use "https://wdlsvnit.github.io/paper-extension/" as redirect url)
+    dropboxURL = "https://www.dropbox.com/oauth2/authorize?response_type=code&client_id=4xu6bbav7553wwg&redirect_uri=https://wdlsvnit.github.io/paper-extension/"
```

Comment:
Two space indentation
Declare variable using var 
use clientId variable(which is declared at top) to create URL and don't push your client id.